Library/Homebrew/cask/lib/hbc/system_command.rb

```diff
@@ -56,14 +57,29 @@ def command
 
     private
 
-    attr_reader :executable, :args, :input, :options, :processed_output, :processed_status, :path
+    attr_reader :executable, :args, :input, :options, :processed_output, :processed_status, :env
 
     attr_predicate :sudo?, :print_stdout?, :print_stderr?, :must_succeed?
 
+    def uppercase_env
```

Comment:
I think this is one abstraction layer too much. I'd just use the actual variable name as a string instead of a symbol. Also makes it easier to grep.

Reply:
Fair point. Done!